src/devices/schneider_electric.ts

```diff
@@ -1028,26 +1028,17 @@ const definitions: Definition[] = [
     },
     {
         zigbeeModel: ['FLS/SYSTEM-M/4'],
-        model: 'WDE002906',
+        model: 'WDE002906/MEG5001-0300',
         vendor: 'Schneider Electric',
         description: 'Wiser wireless switch 1-gang or 2-gang',
         fromZigbee: [fz.command_on, fz.command_off, fz.command_move, fz.command_stop, fz.battery],
```

Comment:
I expect you can completely drop `fromZigbee`

Reply:
yes i removed from and to Zigbee